crates/biome_service/src/workspace.rs

```diff
@@ -116,16 +116,17 @@ impl FileFeaturesResult {
             .is_some_and(|file_name| FileFeaturesResult::PROTECTED_FILES.contains(&file_name))
     }
 
-    /// By default, all features are not supported by a file.
-    const WORKSPACE_FEATURES: [(FeatureName, SupportKind); 3] = [
+    /// By default, only search is supported.
+    const WORKSPACE_FEATURES: [(FeatureName, SupportKind); 4] = [
         (FeatureName::Lint, SupportKind::FileNotSupported),
         (FeatureName::Format, SupportKind::FileNotSupported),
         (FeatureName::OrganizeImports, SupportKind::FileNotSupported),
+        (FeatureName::Search, SupportKind::Supported),
```

Comment:
The default value **must** be `FileNotSupported`. Biome inspects all files, even the ones that can't handle.